packages-ui/roosterjs-react/package.json

```diff
@@ -15,5 +15,6 @@
         "react-dom": ">=16.0.0",
         "@fluentui/react": ">=8.0.0"
     },
-    "main": "./lib/index.ts"
+    "main": "./lib/index.ts",
+    "module": "./lib-mjs/index.ts"
```

Comment:
No need to change these lines. Let normalize.js handle it for you.

Reply:
These changes just add a comma to line 18 and then add module (line 19). I think it's probably good form to have module in here as well instead of inserting via normalize. wdyt?